enterprise/server/util/cpuset/cpuset.go

```diff
@@ -75,60 +75,58 @@ func Format[I constraints.Integer](cpus ...I) string {
 	return strings.Join(cpuStrings, ",")
 }
 
-func parseListFormat(s string) ([]cpuInfo, error) {
-	// Example: "0-1,3" is parsed as []int{0, 1, 3}
-	var nodes []cpuInfo
-	var physicalID int
+// parseCPUs parses a string like "0,1-2" or "0:0-127,1:128-255" to cpuInfos.
+//
+// This string is a comma-separated list of "NODE:CPU_RANGE" pairs where the
+// "NODE:" prefix is optional.
+//
+// If the NODE isn't set, physicalIDs is used to map processor ids to physical
+// ids. Otherwise, the returned CPUs will have physicalID set to -1.
+func parseCPUs(s string, physicalIDs map[int]int) ([]cpuInfo, error) {
```

Comment:
it seems like you could do this re-mapping outside of this function and it would be simpler -- is there a reason to combine them?

Reply:
Done